src/azure_devtools/scenario_tests/preparers.py

```diff
@@ -129,21 +122,39 @@ def process_response(self, response):
         return response
 
 
-# Function wise, enabling large payload recording has nothing to do with resource preparers
-# We still base on it so that this decorator can chain with other preparers w/o too much hacks
-class AllowLargeResponse(AbstractPreparer):
+class AllowLargeResponse(object):  # pylint: disable=too-few-public-methods
```

Comment:
Take this out of `preparer.py`. It is not a preparer since it doesn't create nor delete the resource. Move it to `decorators`